### PR TITLE
WIP: sentence level constraints

### PR DESCRIPTION
**Pull request by kordjamshidi**
add sentence level constraints
not finished yet

### CODE REVIEW

File: saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/SentenceLevelConstraints.scala [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  */
+object SentenceLevelConstraints {
+  val integrityTR = ConstrainedClassifier.constraint[Sentence] {
+    var a: FirstOrderConstraint = null
+    s: Sentence =>
+      a = new FirstOrderConstant(true)
+      (sentences(s) ~> sentenceToToken <~ relationToFirstArgument).foreach
+      {
+        x=>
+        a= a and (((TrajectorPairClassifier on x) is "TR_SP") ==>
```
Comment (Rahgooy) [inline saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/SentenceLevelConstraints.scala:20]: it should be `"TR-SP"`

File: saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/SentenceLevelConstraints.scala [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      a
+  }
+
+  val integrityLM = ConstrainedClassifier.constraint[Sentence] {
+    var a: FirstOrderConstraint = null
+    s: Sentence =>
+      a = new FirstOrderConstant(true)
+      (sentences(s) ~> sentenceToToken <~ relationToFirstArgument).foreach{
+        x=>
+      ((LandmarkPairClassifier on s) is "LM_SP") ==>
```
Comment (Rahgooy) [inline saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/SentenceLevelConstraints.scala:33]: `"LM-SP"`

Comment (kordjamshidi) [reply saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/SentenceLevelConstraints.scala:33]: ah, thanks for the point. That is why probably they did not change the result!!

**Comment by kordjamshidi** [comment]
ah you merged it, thanks. Though it was not complete.

**Comment by Rahgooy** [comment]
ah, sorry, I thought it was.

**Pull request merged**